controllers/invitation.go

```diff
@@ -23,7 +25,7 @@ var invitation *template.Template
 
 func init() {
 	var err error
-	invitation, err = template.ParseFiles("./mail/template.invitation")
+	invitation, err = template.ParseFiles(path.Join(os.Getenv("GOPATH"), "src/github.com/coduno/api/mail/template.invitation"))
```

Comment:
While running with the old path the file can't be found(./mail/template.subscription: no such file or directory).